stripe/test/resources/test_customers.py

```diff
@@ -234,20 +234,38 @@ def test_legacy_delete_subscription(self):
             None
         )
 
+    def test_list_customer_subscriptions(self):
+        customer = stripe.Customer.construct_from({
+            'id': 'cus_foo',
+            'subscriptions': {
+                'object': 'list',
+                'url': '/v1/subscriptions',
```

Comment:
This isn't an accurate test case. The customer subscriptions endpoint will continue to return the same URL: `/v1/customers/cus_foo/subscriptions'